src/device-api/v2.ts

```diff
@@ -421,25 +424,34 @@ router.get('/v2/containerId', async (req: AuthorizedRequest, res) => {
 router.get('/v2/state/status', async (req: AuthorizedRequest, res) => {
 	const appIds: number[] = [];
 	const pending = deviceState.isApplyInProgress();
+	const allImages = await images.getState();
 	const containerStates = (await serviceManager.getAll())
 		.filter((service) => req.auth.isScoped({ apps: [service.appId] }))
 		.map((svc) => {
 			appIds.push(svc.appId);
-			return _.pick(
+			const s = _.pick(
 				svc,
 				'status',
 				'serviceName',
 				'appId',
-				'imageId',
 				'serviceId',
+				'imageId',
 				'containerId',
 				'createdAt',
 			);
+
+			// QUESTION: what to do if we cannot find the image?
+			// this will happen in local mode or if the image database got deleted
```

Comment:
This makes that endpoints behave differently on local mode vs in production, that is not ideal as it means you cannot rely on these endpoints during development. I’ll take a look at improving local mode before merging this PR